### PR TITLE
JBPM-6054 - Create custom task assignments marshalling tests

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
@jomarko could you please review? Thanks

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/utils/Bpmn2Loader.java [hunk 1]
```diff
@@ -30,7 +31,17 @@ public Bpmn2Loader(Class testClass) {
 
     public Definitions loadProcessFromJson(String fileName,
                                            String zOrderEnabled,
-                                           String bpsimDisplay) throws Exception {
+                                           String bpsimDisplay,
+                                           List<String> workitemNames) throws Exception {
+
+        List<String> baseWorkItemNames = new ArrayList<String>(Arrays.asList("Email",
```
Comment (jomarko) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/utils/Bpmn2Loader.java]: Please store this as private static final and reuse on both places from this PR.

Comment (tsurdilo) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/utils/Bpmn2Loader.java]: @jomarko sure but cant be final as its being added upon with passed in ones.

**Comment by tsurdilo** [comment]
@jomarko updated. thanks!

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/utils/Bpmn2Loader.java [hunk 2]
```diff
@@ -24,13 +25,28 @@
...
         this.testClass = testClass;
     }
 
     public Definitions loadProcessFromJson(String fileName,
                                            String zOrderEnabled,
-                                           String bpsimDisplay) throws Exception {
+                                           String bpsimDisplay,
+                                           List<String> workitemNames) throws Exception {
+
+        List<String> workitemNamesList = new ArrayList<>();
```
Comment (jomarko) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/utils/Bpmn2Loader.java]: I think there is no need for this variable:
```java
if (workItemNames == null) {
    workItemNames = new ArrayList<> (baseWorkItemNames);
} else {
    workItemNames.addAll(baseWorkItemNames);
}
```

Please notice that `workitemNames` should be by java conventions `workItemNames`. Sorry that I didn't hint you this in first round of review.

**Comment by tsurdilo** [comment]
@jomarko thanks :) done

**Pull request merged**